.travis.yml

```diff
@@ -1,14 +1,52 @@
 sudo: required
 
-language: java
+language: scala
 
 services:
   - docker
 
+# An alternative approach is to build a full matrix.
+#scala:
+#  - "2.10"
+#  - 2.11
+#
+#env:
+#  - KAFKA_VERSION=0.8.2.2
+#  - KAFKA_VERSION=0.9.0.1
+#  - KAFKA_VERSION=0.10.2.1
+#  - KAFKA_VERSION=0.11.0.2
+#  - KAFKA_VERSION=1.0.1
+#  - KAFKA_VERSION=1.1.0
+
+# This version will be also tagged as 'latest'
+env:
+  global:
+    - LATEST="2.11-1.1.0"
+
+# Build recommended versions based on: http://kafka.apache.org/downloads
```

Comment:
While getting familiar with this setup and to keep build times relatively low and not use up a lot of travis-ci resources, I've stuck to the main major versions of each release along with the recommended scala version.

If we go down the official image route, then we should probably support different architectures as well - so I thought this should also be considered before we end up building 100's of images with every change :)